src/utils/constants.ts

```diff
@@ -1,14 +1,14 @@
-let API_VERSION = "v8";
-
-export const baseEndpoints = {
-  /** Although, the version can be defaulted, keep the v6 as it can be changed to test newer versions when necessary. */
-  BASE_URL: `https://discord.com/api/${API_VERSION}`,
+// These will never be modified and remain constants
+export const discordAPIURLS = {
+  BASE_URL: `https://discord.com/api/v8`,
   CDN_URL: "https://cdn.discordapp.com",
 };
 
-export function changeAPIVersion(number = 7) {
-  API_VERSION = `v${number}`;
-}
+// This can be modified by big brain bots and use a proxy
+export const baseEndpoints = {
```

Comment:
```suggestion
export let baseEndpoints = {
```

Reply:
I don't think this needs to be a let is not reassigned but the properties are mdified later on directly allowing it to remain const